### PR TITLE
Improved the README (again)

### PR DESCRIPTION
**Pull request by RaptaG**
Signed-off-by: RaptaG <george-raptis@tutamail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,35 +30,48 @@
...
+  - TWRP Recovery:
+    - [TWRP recovery](https://twrp.me/Devices)
+  - Optional Addons:
+    - There are different packages of *Google Apps* available:
+      - [MindTheGapps](https://wiki.lineageos.org/gapps#downloads)
+      - [NikGApps](https://nikgapps.com)
+    - [MicroG](https://microg.org)
+      - The recommended way to install MicroG is to use the zip file provided here: [https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
+    - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
+  - or you can just search the web or the [XDA Developers forum](https://forum.xda-developers.com) for an appropriate version for your device.
```
Comment (MagicLike) [inline README.md:70]: The XDA Forums URL has changed to `https://xdaforums.com`

**Review by MagicLike — CHANGES_REQUESTED** [review]
Otherwise looking good!

File: README.md (hunk 1 above)
Comment (RaptaG) [reply README.md]: Fixed

**Review by MagicLike — APPROVED** [review]
LGTM

**Review by tsterbak — APPROVED** [review]
Looks good :)

File: README.md [hunk 2]
```diff
@@ -30,35 +30,48 @@
...
-    - Optional Addons:
-      - There are different packages of *Google Apps* available.
-        - [MindTheGapps](https://wiki.lineageos.org/gapps#downloads)
-        - [NikGApps](https://nikgapps.com)
-      - [MicroG](https://microg.org)
-        - The recommended way to install MicroG is to use the zip file provided here: [https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
-      - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
-    - or you can just search the web or the [xda-developers forum](https://forum.xda-developers.com) for an appropriate version for your device.
-3. Start the desktop app and follow the instructions.
+Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 LTS). Windows and macOS are also supported, but you might experience more issues.<br />
```
Comment (tsterbak) [inline README.md:40]: I don't like the linebreak here, I think it adds not too much.

File: README.md [hunk 3]
```diff
@@ -30,35 +30,48 @@
...
+So far there is no support for ARM-based systems.
+
+### 1. Download OpenAndroidInstaller
+
+Download the correct version for your OS [from the Releases](https://github.com/openandroidinstaller-dev/openandroidinstaller/releases).
```
Comment (tsterbak) [inline README.md:45]: There are versions for Linux, Windows and MacOS (without ARM, maybe this can go here). Also the "(You might need to change permissions to run the executable)" needs to be here.

File: README.md [hunk 4]
```diff
@@ -30,35 +30,48 @@
...
+
+For Linux users, you can get the [official Flatpak version from Flathub](https://flathub.org/apps/org.openandroidinstaller.OpenAndroidInstaller). (You might need to change permissions to run the executable)
```
Comment (tsterbak) [inline README.md:47]:
```suggestion
For Linux users, you can also get the [official Flatpak version from Flathub](https://flathub.org/apps/org.openandroidinstaller.OpenAndroidInstaller).
```

File: README.md [hunk 5]
```diff
@@ -30,35 +30,48 @@
...
+
+For Windows users, you might need to [install the Universal USB Drivers](https://adb.clockworkmod.com) and other potentially drivers needed for your device.
```
Comment (tsterbak) [inline README.md:49]: Maybe this whole section could be structured by OS? Then the information needed for each OS can be grouped together properly?

**Review by tsterbak — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (RaptaG) [reply README.md:49]: I like the idea, will implement it soon

**Comment by MagicLike** [comment]
Hey @RaptaG, would it be possible to make the requested changes, so that it can get merged into the current dev and pulled over to main with the upcoming release?

**Comment by RaptaG** [comment]
> Hey @RaptaG, would it be possible to make the requested changes, so that it can get merged into the current dev and pulled over to main with the upcoming release?

Alright, I'll apply everything now!

**Comment by RaptaG** [comment]
@tsterbak could you not name the release files with their release number so that I may include them like this:

https://github.com/openandroidinstaller-dev/openandroidinstaller/releases/download/latest/openandroidinstaller-ubuntu-latest.zip

https://github.com/openandroidinstaller-dev/openandroidinstaller/releases/download/latest/openandroidinstaller-macos-latest.zip

https://github.com/openandroidinstaller-dev/openandroidinstaller/releases/download/latest/openandroidinstaller-windows-latest.zip

**Comment by RaptaG** [comment]
If you have any other suggestions lmk!
Also, I'd love some help with resolving the merge conflicts..

**Comment by tsterbak** [comment]
> @tsterbak could you not name the release files with their release number so that I may include them like this:

No, I want to release number in it to make it clear which release it is and to be able to link to different files easily. Just link to the release page, that should be fine.

**Comment by tsterbak** [comment]
> If you have any other suggestions lmk!

I will have a look and let you know :) Thank you!

> Also, I'd love some help with resolving the merge conflicts..

I can try to resolve them :)

**Comment by MagicLike** [comment]
> > Also, I'd love some help with resolving the merge conflicts..
> 
> I can try to resolve them :)

Already done! :D

File: README.md [hunk 6]
```diff
@@ -30,35 +30,63 @@
...
-        - The recommended way to install MicroG is to use the zip file provided here: [https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
-      - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
-    - or you can just search the web or the [xda-developers forum](https://xdaforums.com) for an appropriate version for your device.
-3. Start the desktop app and follow the instructions.
+Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 LTS). Windows and macOS are also supported, but you might experience more issues.
+
+### 1. Download OpenAndroidInstaller
+
+> [!WARNING]
+> ARM-based systems are **unsupported**.
```
Comment (MagicLike) [inline README.md:45]: `unsupported` does not sound right, maybe `not supported` would be better and amplify the unsupported status?

File: README.md [hunk 7]
```diff
@@ -30,35 +30,63 @@
...
+
+Follow the instructions for your desktop OS. The executables are compressed inside `.zip` files, so you'll have to extract them first (make sure you have extracting software installed). If you get prompted to, you'll have to adjust the permission of the executable to ensure its proper functionality.
```
Comment (MagicLike) [inline README.md:47]: I don't think, that `desktop OS` needs to be specified, just `OS` should be alright...

File: README.md [hunk 8]
```diff
@@ -30,35 +30,63 @@
...
+> [!WARNING]
+> ARM-based systems are **unsupported**.
+
+Follow the instructions for your desktop OS. The executables are compressed inside `.zip` files, so you'll have to extract them first (make sure you have extracting software installed). If you get prompted to, you'll have to adjust the permission of the executable to ensure its proper functionality. 
+
+#### Linux
+
+You can download OAI from either:
+
+1. From Flathub: [`org.openandroidinstaller.OpenAndroidInstaller`](https://flathub.org/apps/org.openandroidinstaller.OpenAndroidInstaller) (recommended)
```
Comment (MagicLike) [inline README.md:53]: `From` is double worded, as it would read like "You can download OAI from either from Flathub [...]" and should be therefore removed.
I think the `either` should also be removed and the download options would better be in a unsorted list.

File: README.md [hunk 9]
```diff
@@ -30,35 +30,63 @@
...
+
+You can download OAI for Windows from the GitHub Releases: [`openandroidinstaller-windows.zip`](https://github.com/openandroidinstaller-dev/openandroidinstaller/releases/download/latest/openandroidinstaller-windows.zip)
+
+You might also need to [install the Universal USB Drivers](https://adb.clockworkmod.com) and other potentially drivers needed for your device.
+
+#### macOS
+
+For macOS, you can get OpenAndroidInstaller from the GitHub Releases: [`openandroidinstaller-macos.zip`](https://github.com/openandroidinstaller-dev/openandroidinstaller/releases/download/latest/openandroidinstaller-macos.zip)
+
+### 2. Download the custom ROM image & the TWRP recovery image (& optionally addons)
```
Comment (MagicLike) [inline README.md:66]: I would rephrase the header like this: `Download the custom ROM, recovery image & optional addons`, as support for multiple recoveries is coming up.

File: README.md [hunk 10]
```diff
@@ -30,35 +30,63 @@
...
+
+Here are the official links for:
+
+  - some custom ROMs:
```
Comment (MagicLike) [inline README.md:70]: [GrapheneOS](https://grapheneos.org), [CalyxOS](https://calyxos.org) and [DivestOS](https://divestos.org) should also be added

File: README.md [hunk 11]
```diff
@@ -30,35 +30,63 @@
...
+    - [LineageOS](https://wiki.lineageos.org/devices)
+    - [/e/OS](https://doc.e.foundation/devices)
+    - [LineageOS for MicroG](https://download.lineage.microg.org)
+    - [BlissRoms](https://blissroms.org)
+    - [PixelExperience](https://download.pixelexperience.org)
+  - TWRP Recovery:
```
Comment (MagicLike) [inline README.md:76]: This should just be named `Recovery`

File: README.md [hunk 12]
```diff
@@ -30,35 +30,63 @@
...
+      - [MindTheGapps](https://wiki.lineageos.org/gapps#downloads)
+      - [NikGApps](https://nikgapps.com)
+    - [MicroG](https://microg.org)
+      - The recommended way to install MicroG is from the `.zip` file provided here: [`MinMicroG-abuse-CI/releases`](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
+    - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
+  - or you can just search the web or the [XDA Developers forum](https://xdaforums.com) to find an appropriate version for your device.
+
+### 3. Start the desktop app
+
+There, you will be prompted to plug your device to your computer and you will be given a step-by-step tutorial on how to install the ROM, TWRP and the addons for your specific model.
```
Comment (MagicLike) [inline README.md:89]: Just a rephrasing including a grammar correction:
`After starting the app you will be prompted to plug your device into your computer and you will be given a step-by-step tutorial on how to install your new OS for your specific device.`

File: README.md [hunk 13]
```diff
@@ -30,35 +30,63 @@
...
+  - Optional Addons:
+    - There are different packages of *Google Apps* available:
+      - [MindTheGapps](https://wiki.lineageos.org/gapps#downloads)
+      - [NikGApps](https://nikgapps.com)
+    - [MicroG](https://microg.org)
+      - The recommended way to install MicroG is from the `.zip` file provided here: [`MinMicroG-abuse-CI/releases`](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
+    - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
+  - or you can just search the web or the [XDA Developers forum](https://xdaforums.com) to find an appropriate version for your device.
+
+### 3. Start the desktop app
```
Comment (MagicLike) [inline README.md:87]: I think `Start OpenAndroidInstaller` would fit better...

**Review by MagicLike — CHANGES_REQUESTED** [review]
Generally a really nice upgrade for the README, but please consider implementing my requested changes!

**Comment by anon1892** [comment]
Little improvement, I think it can be done here : can Xiaomi devices be sorted? (even it's somewhat hard ^^)
Group “Redmi Note” together, “Redmi ...A”, “Poco”, “Mi” too... And maybe add "Poco” to the brand header, like “Xiaomi / Poco”?

**Comment by RaptaG** [comment]
I'll try to apply everything soon @MagicLike

**Comment by RaptaG** [comment]
I'll add them later today

**Comment by RaptaG** [comment]
> Little improvement, I think it can be done here : can Xiaomi devices be sorted? (even it's somewhat hard ^^) Group “Redmi Note” together, “Redmi ...A”, “Poco”, “Mi” too... And maybe add "Poco” to the brand header, like “Xiaomi / Poco”?

I didn't categorize them though I added `Poco` on the header

**Comment by RaptaG** [comment]
Everything is ready now @MagicLike

**Comment by MagicLike** [comment]
> > Little improvement, I think it can be done here : can Xiaomi devices be sorted? (even it's somewhat hard ^^) Group “Redmi Note” together, “Redmi ...A”, “Poco”, “Mi” too... And maybe add "Poco” to the brand header, like “Xiaomi / Poco”?
> 
> I didn't categorize them though I added `Poco` on the header

The tables need a rework at some point anyway, so it shouldn't be an issue rn :+1:

**Review by MagicLike — APPROVED** [review]
LGMT! - I think this can get merged, other changes can still be done in the future :+1:

**Comment by MagicLike** [comment]
@tsterbak Can you take a look if this on can finally be merged?

**Comment by tsterbak** [comment]
I requested some changes, otherwise it looks good :)

**Comment by tsterbak** [comment]
Any updates on this? I would love to get this merged :blush:

**Comment by MagicLike** [comment]
> Any updates on this? I would love to get this merged 😊

afaik it is finished and just waiting for the merge...

**Review by tsterbak — CHANGES_REQUESTED** [review]

File: README.md [hunk 14]
```diff
@@ -30,35 +30,66 @@
...
-        - [MindTheGapps](https://wiki.lineageos.org/gapps#downloads)
-        - [NikGApps](https://nikgapps.com)
-      - [MicroG](https://microg.org)
-        - The recommended way to install MicroG is to use the zip file provided here: [https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
-      - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
-    - or you can just search the web or the [xda-developers forum](https://xdaforums.com) for an appropriate version for your device.
-3. Start the desktop app and follow the instructions.
+Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 LTS). Windows and macOS are also supported, but you might experience more issues.
+
+### 1. Download OpenAndroidInstaller
```
[the remark this answers is not in the document]
Comment (RaptaG) [reply README.md:42]: Alright, I'll fix these 2 things you have pointed out and this should be ready to be merged

**Comment by MagicLike** [comment]
Any updates on this one @RaptaG?

**Comment by RaptaG** [comment]
> Any updates on this one @RaptaG?

Oh, right, I forgot about this one
Ι **will** finish it tomorrow, at about 16:30 GMT+2

**Comment by RaptaG** [comment]
@tsterbak @MagicLike how does it look now?

**Comment by RaptaG** [comment]
Also, could I get some help resolving the conflicts?

**Review by tsterbak — APPROVED** [review]
